src/tests/Feature/QuizTest.php

```diff
@@ -39,6 +40,52 @@ private function createAdmin()
         return $user;
     }
 
+    private function createRolePermission() {
```

Comment:
Duplicate code, you can run seeders directly from tests, doing it this way works but isn't very clean

Reply:
Yes, I saw that call seeder way, but in that BouncerSeeder, we created admin user, and some permissions, but I saw in this test the createAdmin is a separated method for some reason so I don't want to mess this up and only added the necessary setup data there.